README.md

```diff
@@ -40,7 +38,40 @@ and the requisite tables will be created.
 
 ## Usage
 
-TODO: Write usage instructions here
+Add your Pivotal Tracker API token to your secrets.yml:
+
+`tracker_api_token: <%= ENV["PIVOTAL_TRACKER_API_TOKEN"] %>`
+
+Create a new rake task called `import.rake`
```

Comment:
Okay, finished this up and PR'd it!
https://github.com/foraker/active_pivot/pull/2